projects/example-app/src/app/app.module.ts

```diff
@@ -55,7 +54,9 @@ import { AppRoutingModule } from '@example-app/app-routing.module';
      */
     StoreDevtoolsModule.instrument({
       name: 'NgRx Book Store App',
-      logOnly: environment.production,
```

Comment:
I would match this instead  https://github.com/ngrx/platform/blob/master/modules/schematics/src/store/index.ts#L102. That way you still have it during development

Reply:
But then the deployed version at https://ngrx.github.io/platform/example-app/#/login won't have the devtools enabled since it's a production build? The main reason for this change would be to have it enabled on the deployed example app, which is included in the redux devtools docs.

The maintainer of redux devtools mentioned he would write e2e tests on the extension used in our example app.

Would it be OK to mention the snippet as a comment? Or should we indeed not include the store devtools?

Reply:
I still want to include it as not something we have to flip manually when deploying the example. If we need to deploy it with the devtools enabled, we can do that using configuration and the environment variables.

Reply:
Oh yea definitely! I also don't want to introduce a (manual) toggle on builds.
I wanted to always build the app with the devtools fully enabled but still wanted to mention that as a user, you wouldn't want to include the dev tools in a production build.